graphql/http_handler.go

```diff
@@ -38,6 +38,17 @@ func Handler(apiURL string) func(w http.ResponseWriter, r *http.Request) {
 	srv.SetRecoverFunc(func(ctx context.Context, err interface{}) error {
 		queryPath := graphql.GetFieldContext(ctx).Path()
 
+		if queryPath == nil {
+			grip.Error(message.Fields{
```

Comment:
We're already logging critical right below (which I don't love re the info below but alas) and it already logs what queryPath is (if you want to really know if it's nil and not just empty you could add a `"query_is_nil": queryPath==nil` below)

Reply:
removed!